core/typeguards.ts

```diff
@@ -0,0 +1,60 @@
+import is from '@sindresorhus/is';
```

Comment:
Codacy has a fix for the issue: [Replace `'@sindresorhus/is'` with `"@sindresorhus/is"`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7711626)
```suggestion
import is from "@sindresorhus/is";
```